From d9ca591806cbb3ec561caa961705128a061abe5c Mon Sep 17 00:00:00 2001
From: Boyan Vladinov <nayobix@nayobix.org>
Date: Wed, 21 Jan 2015 14:24:59 +0200
Subject: [PATCH] Add new Hardware HAL - SensorHubMgmt - [3/3]

Signed-off-by: Boyan Vladinov <nayobix@nayobix.org>
---
 service_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/service_contexts b/service_contexts
index 8585fcf..c7cfe88 100644
--- a/service_contexts
+++ b/service_contexts
@@ -120,5 +120,6 @@ wifip2p                                   u:object_r:system_server_service:s0
 wifiscanner                               u:object_r:system_server_service:s0
 wifi                                      u:object_r:system_server_service:s0
 window                                    u:object_r:system_server_service:s0
+sensorhubmgmt                             u:object_r:system_server_service:s0
 
 *                                         u:object_r:default_android_service:s0
-- 
2.0.0

